packages/adapter-node/src/server.js

```diff
@@ -7,51 +7,55 @@ import { URL, fileURLToPath } from 'url';
 // eslint-disable-next-line import/no-unresolved
 import { get_body } from '@sveltejs/kit/http';
 // App is a dynamic file built from the application layer.
-/*eslint import/no-unresolved: [2, { ignore: ['\.\/app\.js$'] }]*/
-import * as app from './app.js';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
-
-const { PORT = 3000 } = process.env; // TODO configure via svelte.config.js
-
-const mutable = (dir) =>
-	sirv(dir, {
-		etag: true,
-		maxAge: 0
-	});
-
 const noop_handler = (_req, _res, next) => next();
-
-const prerendered_handler = fs.existsSync('prerendered') ? mutable('prerendered') : noop_handler;
-
-const assets_handler = sirv(join(__dirname, '/assets'), {
-	maxAge: 31536000,
-	immutable: true
-});
-
-polka()
-	.use(compression({ threshold: 0 }), assets_handler, prerendered_handler, async (req, res) => {
-		const parsed = new URL(req.url || '', 'http://localhost');
-		const rendered = await app.render({
-			method: req.method,
-			headers: req.headers, // TODO: what about repeated headers, i.e. string[]
-			path: parsed.pathname,
-			body: await get_body(req),
-			query: parsed.searchParams
+const paths = {
+	assets: join(__dirname, '/assets'),
+	prerendered: 'prerendered'
```

Comment:
not totally sure why one of these is absolute and one is relative — feels like it might be unintentional. though i don't know which is correct off the top of my head

Reply:
fixed to both be use the join + __dirname approach to be consistent.